### PR TITLE
Bt vod mpx

### PR DESCRIPTION
**Pull request by wwajerowicz**

### CODE REVIEW

File: src/main/java/org/atlasapi/remotesite/btvod/BtVodMpxImageExtractor.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
+package org.atlasapi.remotesite.btvod;
```
Comment (mbst-jamie) [inline src/main/java/org/atlasapi/remotesite/btvod/BtVodMpxImageExtractor.java]: I got to here

File: pom.xml [hunk 2]
```diff
@@ -335,6 +335,11 @@
     <dependencies>
         <dependency>
             <groupId>org.atlasapi</groupId>
+            <artifactId>atlas-model</artifactId>
```
Comment (tfm) [inline pom.xml]: My earlier comment about this has been lost. Why have we added this dependency?

File: src/main/java/org/atlasapi/remotesite/btvod/model/BtVodPlproduct$pricingTier.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
+package org.atlasapi.remotesite.btvod.model;
+
+public class BtVodPlproduct$pricingTier {
+
+    private Long plproduct$absoluteStart;
+    private Long plproduct$absoluteEnd;
+    private BtVodPlproduct$amounts plproduct$amounts;
+
+    public BtVodPlproduct$pricingTier() {
```
Comment (tfm) [inline src/main/java/org/atlasapi/remotesite/btvod/model/BtVodPlproduct$pricingTier.java]: We were going to rename these hideous fields/classes with $ in them

File: src/main/resources/config/environments/stage.properties [hunk 4]
```diff
@@ -59,6 +59,7 @@ messaging.system=AtlasOwlStage
 
 
 bt.events.s3.bucket=stage-events-data
+bt.vod.mpx.feed.baseUrl=http://feed.product.theplatform.eu/f/CiIRPC/btv-prd-search
```
Comment (tfm) [inline src/main/resources/config/environments/stage.properties]: Remove this, it should be in puppet

File: src/main/resources/config/environments/prod.properties [hunk 5]
```diff
@@ -58,6 +58,7 @@ messaging.system=AtlasOwlProd
 
 
 bt.events.s3.bucket=prod-events-data
+bt.vod.mpx.feed.baseUrl=http://feed.product.theplatform.eu/f/CiIRPC/btv-prd-search
```
Comment (tfm) [inline src/main/resources/config/environments/prod.properties]: Remove this, it should be in puppet

**Comment by tfm** [comment]
@wwajerowicz back to you

File: pom.xml (hunk 2 above)
Comment (wwajerowicz) [inline pom.xml]: I think I had some problems updating it getting updated from my local maven. I think having an explicit dependency also makes it a bit clearer that it acutally depends on model.

**Pull request merged**